src/modules/tenant/TenantDropdown.js

```diff
@@ -83,7 +94,7 @@ class TenantLabel extends Component {
 const mapStateToProps = state => {
 	return {
 		current: state.tenant.current,
-		tenants: state.tenant.tenants,
+		tenants: state?.auth?.userinfo.tenants ? state?.auth?.userinfo.tenants : state.tenant.tenants,
```

Comment:
`state.tenant.tenants` should be filled from `state?.auth?.userinfo.tenant` after receiving the response from userinfo.
So this line in incorrect.

Reply:
@ateska So should I use userinfo API call in tenant module? And since it is used in auth module, would it make sense to move this call to Application.js and dispatch the response from there (to get userinfo in tenant & auth module without separate calls)?